frontend/src/hooks/useFetchQuestionById.js

```diff
@@ -0,0 +1,31 @@
+import axios from '../api/axios';
+import {useState, useEffect} from 'react'
+import { URL_RETRIEVE_QUESTION_ID } from '../constants';
+
+const useFetchQuestionById = (questionId) => {
+    const [loading, setLoading] = useState(true);
+    const [data, setData] = useState([])
+  
+    useEffect((questionId) => {
```

Comment:
```suggestion
    useEffect(() => {
```
I believe the function called by `useEffect` doesn't take any argument, and `questionId` is already passed to the outer hook so it should be ok to not pass the value here.

Reply:
Oh I tried that, it somehow fails the build, not too sure why this is an issue

Reply:
Was it an eslint build warning? I think it could be the useEffect dependency array, may need to use `[questionId]` instead of `[]`, if we don't pass in the questionId.

Reply:
Solved! Thanks